api/admin/package.json

```diff
@@ -9,6 +9,6 @@
   "author": "NYPL",
   "license": "Apache-2.0",
   "dependencies": {
-    "simplified-circulation-web": "0.0.14"
+    "simplified-circulation-web": "0.0.15"
```

Comment:
This version will be published when I finish https://github.com/NYPL-Simplified/circulation-web/issues/77.

We shouldn't merge until then.